TransportServerWSS.js

```diff
@@ -1,17 +1,21 @@
-class TransportServerWS {
+class TransportServerWSS {
```

Comment:
Let's rename this file. Really confusing naming: first thing that comes in - web socket secure, not web socket server